### ISSUE TITLE
Creative control plugins unable to handle PS stones Build 10.2

### ISSUE THREAD

**Issue by frizzbee30** [marcelo-mason/PreciousStones#963]
Hi, we have been trying an alternative creative control plugin to the now outdated Fuminghara one.

Besides the strange issue already raised, where the block has to be placed twice and the meta-name **must** also autoset for the field to register, there is still an issue with creative control and PS stones.

If any PS stone assigned block is placed in creative mode, it can be broken in creative mode and drops, this is not the case with non PS stone blocks

These blocks can then be picked up to bypass creative control when the player swaps to 'survival' mode.

It is possible to set the no drop etc on the PS stone after placement, but the field isn't generated, as they have to be placed, broken and placed again to register as a PS field (ie once the meta-name has been set).

It's as if the blocks are in 'limbo' after first being placed, neither ps stones nor regular blocks

If you need any videos etc of this happening, please let me know.

**Comment by marcelo-mason** [comment]
Video w/audio explaining would help.

**Comment by frizzbee30** [comment]
Thanks, I will get one sorted, probably this weekend to demonstrate.

**Comment by frizzbee30** [comment]
Hi , I have made a quick video, not the best but demonstrates the issue. The config is set to block precious stones in creative mode. When I place the gold block (one of our PS stones) and gold ore (regular block) in creative mode, then switch to survival, the gold block drops while the gold ore doesn't. The gold block then adopts the meta data which i am assuming blocks creative control from preventing the drop

[](https://youtu.be/TLMyn5yu23I)


I have tested this on two creative control plugins and both have the same problem with identifying the PS blocks as placed in 'creative' mode. obviously it potentially leaves an exploit for players with GM1 access to place and break blocks, or for others to break blocks that they have placed for them.

This isn't an issue with our current 'live' version of PS stones

Thanks :)
